### PR TITLE
Me too

### PR DESCRIPTION
**Pull request by SwiftLover**
Hyphenated made up words

### CODE REVIEW

**Comment by rShetty** [comment]
:+1:

File: docs/AccessControl.rst [hunk 1]
```diff
@@ -153,8 +153,8 @@ A tuple type's access level is the minimum of the access levels of its
 elements. A function type's access level is the minimum of the access levels of
 its input and return types.
 
-A typealias may have any access level up to the access level of the type it
-aliases. That is, a ``private`` typealias can refer to an ``public`` type, but
+A type-alias may have any access level up to the access level of the type it
```
Comment (kylef) [inline docs/AccessControl.rst]: Isn't `typealias` one word already a Swift keyword? I think this makes sense to leave as-is.

Comment (SwiftLover) [inline docs/AccessControl.rst]: Apples own documentation break it into two words https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Declarations.html

So if anything the hyphen should be removed.

**Comment by jrose-apple** [comment]
Hm. The compiler's not really consistent with the formal docs on this already, and I'm a little iffy about "type alias" sounding like a general term instead of specifically referring to what you get from a standalone `typealias`. I think I'd rather leave this as is.

**Pull request closed without merging**